radio/src/opentx.cpp

```diff
@@ -2572,7 +2572,7 @@ int main()
   // lcdSetRefVolt(25);
 #endif
 
-#if defined(PCBTARANIS) || defined(PCBHORUS)
+#if defined(SPLASH) && (defined(PCBTARANIS) || defined(PCBHORUS))
```

Comment:
Otherwise link error with Horus build:
```
CMakeFiles\opentx-horus-simulator.dir/objects.a(opentx.cpp.obj): In function `Z8simuMainPv':

radio/src/opentx.cpp:2576: undefined reference to `drawSplash()'
```
https://github.com/mpaperno/opentx/blob/e707d9c58e6adfd8762534a9ea98998e234329b4/radio/src/gui/480x272/splash.cpp#L24